Events/PinwheelEventType.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  PinwheelEvent.swift
+//  PinwheelSDK
+//
+//  Created by Robby Abaya on 2/11/21.
+//
+
+import Foundation
+
+public enum PinwheelEventType: String {
+    case open
+    case selectEmployer = "select_employer"
```

Comment:
Minor nit - in Android at least, the enum capitalization standard is all caps with underscores. If that's the same in iOS you should be able to do the same given your values right? Not a deal breaker.

Reply:
Camel casing is the convention in Swift, so I'd like to keep it this way.